DQMServices/Core/src/DQMNet.cc

```diff
@@ -205,11 +205,16 @@ DQMNet::unpackQualityData(QReports &qr, uint32_t &flags, const char *from)
   while (*qdata)
   {
     ++nqv;
-    while (*qdata) ++qdata; ++qdata;
```

Comment:
This actually makes sense if you look at DQMNet::packQualityData. But I agree, this requires at least a comment.